Makefile

```diff
@@ -25,6 +25,7 @@ format:  ## Format the code.
 lint:  ## Run all linters (black/ruff/pylint/mypy).
 	poetry run black --check .
 	poetry run ruff check .
+	poetry run pylint eq_cir_converter_service
```

Comment:
You most likely want to run pylint against python written tests as well, so possibly `poetry run pylint .`? Also, we should consider this https://pylint.pycqa.org/en/stable/user_guide/usage/run.html#parallel-execution

Reply:
Just wondering if it may also be worth adding `--reports=n` to help reduce the verbose output as well?

Reply:
>Just wondering if it may also be worth adding --reports=n to help reduce the verbose output as well?

Can see that the `--reports=n` has been added now 👍